css/bootstrap.css

```diff
@@ -6758,3 +6771,43 @@ button.close {
   }
 }
 /*# sourceMappingURL=bootstrap.css.map */
+
+
+footer
+{
+    background-color:#4A148C;
+    color: #9649CB;
+    clear:both;
+    text-align:left;
+    padding:2%;
```

Comment:
Remove this. This is what's causing the footer to be funky.
Also, use ```box-sizing: content-box``` if it's still being funny.